_data/publications.yml

```diff
@@ -27,6 +27,13 @@
    where: Los Angeles, California, USA, June 25
    pdf: "2025_Tung_RSS_workshop_CRED"
    subteams: [algorithmic hri and human-ai teaming, yi-shiuan, workshop, social intelligence]
+ - title: "Towards Zero-Shot Coordination between Teams of Agents: The N-XPlay Framework"
+   authors: Ava Abderezaei, Chi-Hui Lin, Joseph Miceli, Naren Sivagnanadasan, Stéphane Aroca-Ouellette, Jake Brawer, Alessandro Roncone
+   year: 2025
+   venue: "RSS Workshop on Scalable and Resilient Multi-Robot Systems: Decision-Making, Coordination, and Learning"
+   where: Los Angeles, California, USA, June 25
+   pdf: "2025_Abdrezaei_RSS_workshop_NXPLAY"
+   subteams: [multi-team systems and human-ai teaming, ava, chi-hui, workshop, social intelligence]
```

Comment:
please use `algorithmic hri and human-ai teaming`, it's a specific tag I use for auto generating websites or parts of websites.

Reply:
Also add `naren` and `stephane` plz (all authors that are phd students)